lib/food-service.js

```diff
@@ -17,5 +17,20 @@ class FoodService {
   }
 }
 
+$(".food-form").on('submit', function(e){
+  e.preventDefault();
+  var post_url = $(this).attr("action");
+  var request_method = $(this).attr("method");
+  var form_data = $(this).serialize();
+
+  $.ajax({
+    url: post_url,
+    type: request_method,
+    data: form_data
+  }).done(function(response){
+    $("#server-results").html(response)
+  })
+})
```

Comment:
This is where we could potential fix the exposed url. If you move the ajax (or fetch) call into a method on the food service object, then we would have access to the base url here. Also, it would shorten the event listener. Ultimately, we will probably want to return our event listeners to a single method call that we can add to our index.js